src/bindings/python/tests_compatibility/test_inference_engine/test_InferRequest.py

```diff
@@ -401,6 +403,7 @@ def test_blob_setter(device):
     assert np.allclose(res_1, res_2, atol=1e-2, rtol=1e-2)
 
 
+@pytest.mark.skip(reason="Plugin API 2.0 doesn't support GetPreProcess.")
```

Comment:
during preprocessing insertion into the mode, the information about preprocessing is stored in runtime information of the preprocessed model. We can restore this information, please, refer to template plugin about details how it can be done

Reply:
It seems Template plugin skip some of preprocess test items:
https://github.com/openvinotoolkit/openvino/blob/84d35ab6f92f4786977967ebeb17be8f5ee55e7f/src/plugins/template/tests/functional/skip_tests_config.cpp#L102-L104